### PR TITLE
Component buildout

### PR DESCRIPTION
**Pull request by bbusch19**

### CODE REVIEW

File: components/BaseHeader.vue [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+import BaseHeaderNavigation from '~/components/BaseHeaderNavigation'
+
+export default {
+  components: {
+    BaseHeaderNavigation
+  }
+}
+</script>
+
+<style lang="scss" scoped>
```
Comment (tomgalpin) [inline components/BaseHeader.vue]: two style sections for this component

File: components/BaseHeader.vue [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+      <p>OWL <br /><span>Support Center</span></p>
+    </div>
+    <BaseHeaderNavigation />
+    <div>
+      <p>Icons go here</p>
+    </div>
+  </header>
+</template>
+
+<style lang="scss" scoped>
```
Comment (tomgalpin) [inline components/BaseHeader.vue:24]: two style sections

**Review by tomgalpin — COMMENTED** [review]
Two style sections for Baseheader component

File: components/BaseHeader.vue (hunk 2 above)
Comment (bbusch19) [reply components/BaseHeader.vue:24]: @tomgalpin Good catch. No clue how this happened!

**Comment by tomgalpin** [comment]
lgtm

**Pull request merged**